src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/ManageUsersApiExceptionHandler.kt

```diff
@@ -101,10 +101,10 @@ class HmppsManageUsersApiExceptionHandler {
   }
 
   @ExceptionHandler(ValidEmailException::class)
-  fun handleAuthUserLastGroupException(e: ValidEmailException): ResponseEntity<ErrorResponse> {
+  fun handleValidEmailException(e: ValidEmailException): ResponseEntity<ErrorResponse> {
     log.info("Email validation exception caught: {}", e.message)
     return ResponseEntity.badRequest()
-      .body(ErrorResponse(status = BAD_REQUEST.value(), developerMessage = e.message))
+      .body(ErrorResponse(status = BAD_REQUEST.value(), userMessage = "Validation failure: ${e.message}", developerMessage = e.message))
```

Comment:
for validation errors, current implementation adds errored field with message..
I added new variable to error message to be more specific and better handling in UI , just in case

"uiErrors": [
        {
            "error_description": " Already exists, for field userId with reason: User with username ravindra56@1digital.j.g.uk already exist",
            "field": "userId"
        }
    ]

Reply:
Discussed with @ravmoj and @karenmillermoj - agreed that the existing UI client will not need to parse individual field errors as all are trapped in UI itself before the API is called. @ravmoj agreed to remove the new uiErrors field.